src/ducks/recurrence/queries.js

```diff
@@ -20,11 +20,7 @@ export const bundleTransactionsQueryConn = ({ bundle }) => {
     query: () => {
       const initialQDef = queryRecurrenceTransactions(bundle)
       const qDef = initialQDef
-        .sortBy([
-          { 'relationships.recurrence.data._id': 'desc' },
-          { date: 'desc' }
-        ])
-        .indexFields(['relationships.recurrence.data._id', 'date'])
+        .sortBy([{ date: 'desc' }])
```

Comment:
Pour ma curiosité, comment ça cassait Pouch ?

On a bien l'index sur les mêmes champs que le sortBy donc ça aurait dû fonctionner.

Par contre, ça excluait des réponses tous les documents qui n'avaient pas de data._id. C'était ça le souci ?

Reply:
Non c'était pas ça le soucis, je merge et je check quand je repasse sur mobile, je me suis noté ça.